examples/pytorch/eeg-gcnn/main.py

```diff
@@ -91,6 +92,12 @@
     # set up input and targets from files
     memmap_x = f"psd_features_data_X"
     memmap_y = f"labels_y"
+    if os.path.getsize(memmap_x) == 86528478:
```

Comment:
why 86528478 is so special?

Reply:
In fact, 86528478 is the length of the unaligned `psd_features_data_X`. I agree with you, it would be much better to get rid of this constant.  Because the alignment of files is very fast, I implemented
```
def load_memory_mapped_array(file_name):
    # Since `file_name` is not always byte-aligned,
    # let's fix this to avoid the warning.
    joblib.dump(joblib.load(file_name), file_name)
    return load(file_name, mmap_mode="r")
```
and started doing it all the time, no matter the file size.